web/src/features/panels/zone/FeedbackCard.tsx

```diff
@@ -0,0 +1,270 @@
+import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group';
+import Pill from 'components/Pill';
+import { ChangeEvent, Dispatch, SetStateAction, useState } from 'react';
+import { HiOutlineX } from 'react-icons/hi';
+import { useTranslation } from 'translation/translation';
+
+export default function FeedbackCard({
+  estimationMethod,
+}: {
+  estimationMethod?: string;
+}) {
+  const [isClosed, setIsClosed] = useState(false);
+  const [state, setState] = useState('1');
```

Comment:
I think we should use some proper names here to make it easier to understand what each state means :) 

You could use an enum like we do for `TimeAverages` - this means we're using strings, but in a way where you can't misspell it and it's easier to use (your editor can show you what options are available, etc.)

```ts
enum FeedbackState {
  OPEN = 'open',
  SUCCESS = 'success',
  // etc.
}

// Usage in component
const [state, setState] = useState(FeedbackState.OPEN);

// Usage for type
function FeedbackActions({state}: { state: FeedbackState }) {}
```

Reply:
Good idea. I've made the changes to accommodate this in commit [change state to feedbackState and use enum](https://github.com/electricitymaps/electricitymaps-contrib/pull/6464/commits/ab89aaea7c667f9412162100bd811d9a96da05de)